File: agent-world-lab/backend/python/app/infrastructure/services/baseline_agent_orchestrator.py
from __future__ import annotations

from app.application.interfaces import IAgentOrchestrator


class BaselineAgentOrchestrator(IAgentOrchestrator):
    def generate_reply(self, message: str, cloud_provider: str, agent_mode: str, model_mode: str) -> str:
        return f"[{cloud_provider}/{agent_mode}/{model_mode}] {message}"

File: agent-world-lab/backend/src/AgentWorldLab.Infrastructure/Services/BaselineAgentOrchestrator.cs
using AgentWorldLab.Application.Abstractions;

namespace AgentWorldLab.Infrastructure.Services;

public sealed class BaselineAgentOrchestrator : IAgentOrchestrator
{
    public string GenerateReply(string userMessage, string cloudProvider, string agentMode, string modelMode)
    {
        return $"[{cloudProvider}/{agentMode}/{modelMode}] {userMessage}";
    }
}
